frontend/src/components/6.Countdown/CountDown.js

```diff
@@ -0,0 +1,55 @@
+import {Link} from 'react-router-dom';
+import React, {useState} from 'react';
+import Help from '../Common/Help';
+import '../Common/Help.css';
+import {useHistory} from 'react-router-dom';
+
+const CountDown = () => {
+  const [ButtonPopup, setButtonPopup] = useState(false);
+  const [seconds, setSeconds] = React.useState(3);
+
+  React.useEffect(() => {
+    if (seconds >= 0) {
+      setTimeout(() => setSeconds(seconds - 1), 1000);
+    }
+  });
+
+  const history = useHistory();
+  /**
+   */
+  function goNextPge() {
+    history.push('/Lobby');
+  }
+
+  return (
+    <>
+      <h1 className='Title'>yumble</h1>
+      <div className='MakeCentre'>
+        <div>
+          <div className='StartTitle'>{seconds}</div>
+        </div>
+      </div>
+      <Link to='/Lobby'>
+        <button className='SmallBtn' id='BackButton'>
+          Back
+        </button>{' '}
+      </Link>
+      <button
+        onClick={() => setButtonPopup(true)}
+        className='SmallBtn'
+        id='HelpButton'
+      >
+        help?
+      </button>
+      <Help trigger={ButtonPopup} setTrigger={setButtonPopup}>
+        <p>
+          This is the countdown, please wait for the timer and you be set for
+          your date!
+        </p>
+      </Help>
+      {seconds == -1 ? goNextPge() : null}
```

Comment:
Do we want it to say "Go!" on the last count before going to the next page- just like in the mockups?

Reply:
Added the 'Go!' to CountDown. Had some problems but realised I could just hide the 'Go!' when countdown is not zero haha. Thanks for the feedback jemimahmartinez! Appreciate it!